src/earn/EarnEnterAmount.tsx

```diff
@@ -683,6 +797,29 @@ const styles = StyleSheet.create({
     flexWrap: 'wrap',
     textAlign: 'left',
   },
+  gap8: {
+    gap: Spacing.Smallest8,
+  },
+  bottomSheetTextContent: {
+    marginBottom: Spacing.XLarge48,
+    marginTop: Spacing.Smallest8,
+  },
+  bottomSheetLineLabel: {
+    maxWidth: '50%',
+    textAlign: 'left',
+  },
+  bottomSheetLineLabelText: {
+    maxWidth: '50%',
```

Comment:
I think this causes amounts to overflow on smaller devices with default fonts despite there being enough space in the middle (see screenshot above), maybe we can remove this?

Reply:
what about adjusting the %'s a bit like 40-60? It seems like the amounts would be longer (especially if the fee currency is ETH). The device I used is iPhone 13, which isn't that small, and there was still overflow on it, which doesn't seem great. Although, I am not sure what we should be optimizing for, whether it's default font size or large sizes.

Reply:
Went with 40% & 60% in c2422f406940d19da317429b8759dc50a5a7347a!